src/link/prepQueryLinks.js

```diff
@@ -70,24 +75,38 @@ export default function prepQueryLinks(rootQuery, defs) {
     } else {
       for (const node of query) {
         if (!isBranch(node)) continue;
-        let usedHere = prepQueryDef(
-          node.children,
-          rest,
-          def,
-          {
-            ...vars,
-            [key.slice(1)]: node.key,
-          },
-          node.version,
-        );
-
+        let usedHere;
+        if (node.prefix) {
```

Comment:
`node.prefix: true` indicates that this node is the "filter" part of a filter + pagination query.

In the "plumbing" form of a graffy query / graph, we split filtering and pagination across two levels. Example: `$key: { x: 5, $all: true}` is encoded into a parent node with a key encoding `{x: 5}`, with a child "range" node that encodes `{ $all: true }`. The parent node in this example will have an attribute `prefix` set to true.